test/mailers/account_mailer_test.rb

```diff
@@ -28,47 +28,41 @@ def run_mail_test(name, user = nil)
     if text_files.any?
       user.email_html = false if user
       yield
-      email = ActionMailer::Base.deliveries.last.encoded
-      email = remove_extraneous_email_headers(email)
+      email = String.new(whole_email(space: true))
       assert_string_equal_file(email, *text_files)
     end
 
     return unless html_files.any?
 
     user.email_html = true if user
     yield
-    email = ActionMailer::Base.deliveries.last.encoded
-    email = remove_extraneous_email_headers(email)
-    fix_mac_vs_pc!(email)
+    email = String.new(whole_email(space: false))
```

Comment:
`String.new(string)` is necessary in Ruby 2.7 to create a mutable string. Takes some getting used to... you can't just `string + string` here.